dev/PipsPager/PipsPager_21h1_themeresources.xaml

```diff
@@ -0,0 +1,9 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <x:String x:Key="PipsPagerPreviousPageButtonGlyph">&#xF08D;</x:String>
+    <x:String x:Key="PipsPagerNextPageButtonGlyph">&#xF08F;</x:String>
```

Comment:
&#xF08E;

<x:String x:Key="PipsPagerNextPageButtonGlyph">&#xF08E;</x:String>

Reply:
You are right. I just notice it when I do the screenshot on it